packages/core/template/project/DeveloperGuide.md

```diff
@@ -0,0 +1,285 @@
+---
+  layout: default.md
+  title: "Developer Guide"
+  pageNav: 3
+---
+
+# ProjectEx Developer Guide
+
+<!-- * Table of Contents -->
+<page-nav-print />
+
+--------------------------------------------------------------------------------------------------------------------
+
+## **Acknowledgements**
+
+_{ list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well }_
+
+--------------------------------------------------------------------------------------------------------------------
+
+## **Setting up, getting started**
+
+Refer to the guide [_Setting up and getting started_](SettingUp.md).
+
+--------------------------------------------------------------------------------------------------------------------
+
+## **Design**
+
+<box type="tip">
+This section describes the architecture of your app, explaining how the main components work and interact with each other. Using architecture diagrams is recommended.
+</box>
+
+### Architecture
+
+The ***Architecture Diagram*** given below explains the high-level design of the ProjectEx App.
+
+<img src="images/johndoe.png" width="200px">
```

Comment:
Maybe you can make use of the annotate feature in MarkBind here to put a label on top of the image highlighting that this is just a sample image? I wondering if the image was just not loading at first.